source/SH3/system/glbuffer.cpp

```diff
@@ -0,0 +1,52 @@
+/** @file
+ *
+ *  Implementation of @ref glbuffer.hpp
+ *
+ *  @copyright 2017 Palm Studios
+ *
+ *  @date 8-6-2017
+ *
+ *  @author Jesse Buhagiar
+ */
+#include "SH3/system/glbuffer.hpp"
+#include "SH3/system/log.hpp"
+
+#define GL_VBO_UNBIND 0
+
+using namespace sh3_gl;
+
+void sh3_glbuffer::Create()
+{
+    GLenum err;
```

Comment:
We can define a `SH3ClearGLError` function, which pops all errors of the error queue and warns about them. But this should be in a separate commit. Even in a separate pull request if you want.